src/asynk.rs

```diff
@@ -636,6 +636,14 @@ impl Options {
         }
     }
 
+    /// Select option to include retry logic with backoff 
+    /// on first connect. 
+    pub fn with_retry_on_failed_connect(self) -> Options {
```

Comment:
All other options are spelled without `with`. I would remove it for consistency.

Reply:
Was about to suggest the same thing.

Reply:
Got it.